doc/components.md

```diff
@@ -78,20 +78,58 @@ This component uses an instance of `Svg` class to represent a Component that has
     android.y = 100;
 ```
 
-## FlareAnimation Component
+## FlareActor Component
 
-This component wraps an instance of the [FlareAnimation](/doc/images.md#FlareAnimation), it receives the filename of the Flare animation file, which animation from that file you want to use, and the `width` and `height` of the rendered animation.
+*Note*: The previous implementation of a Flare integration API using `FlareAnimation` and `FlareComponent` has been deprecated.
 
-```dart
-    final fileName = "assets/Bob_Minion.flr";
-    final animation = "Wave";
-    final width = 306;
-    final height = 228;
+To use Flare within Flame, use the [`flame_flare`](https://github.com/flame-engine/flame_flare) package.
+
+This is the interface to use a [flare animation](https://pub.dev/packages/flare_flutter) within flame.
+`FlareActorComponent` has almost the same API as of flare's FlareActor widget. It receives the animation filename (that are loaded by default with `Flame.bundle`),
+it also can receive a FlareController that can play multiple animations and control nodes.
 
-    FlareComponent flareAnimation = FlareComponent(fileName, animation, width, height);
+```dart
+    import 'package:flame_flare/flame_flare.dart';
+
+    // your implementation of FlareController
+    class BurrController extends FlareControls {
+        
+        ActorNode sinnersNode;
```

Comment:
Can't we have more clear name examples here?

Reply:
I'm ok with fun hamilton references but not strongly in favour either, anything would work for me

Reply:
Although I prefer to have more clear things here, as I think it is confusing for me and I bet it will be for more people. This is just suggestions, I have already approved the PR.

Reply:
Changed to a possibly more readable hamilton reference.